src/Serilog.Sinks.Seq/SeqLoggerConfigurationExtensions.cs

```diff
@@ -49,8 +50,9 @@ public static class SeqLoggerConfigurationExtensions
         /// <returns>Logger configuration, allowing configuration to continue.</returns>
         /// <exception cref="ArgumentNullException">A required parameter is null.</exception>
         public static LoggerConfiguration Seq(
-            this LoggerSinkConfiguration loggerSinkConfiguration,
+            this LoggerSinkConfiguration loggerSinkConfiguration,            
             string serverUrl,
+            HttpMessageHandler messageHandler = null,
```

Comment:
This will need to be added as the last parameter in the list to avoid source-breaking changes when people specify all of the arguments sequentially.